docs/modes-of-operation.rst

```diff
@@ -326,3 +326,56 @@ of 99 seconds and the specified string. TXT Records with names starting
 with ``_tstamp.`` get their value (rdata) set to the current time stamp.
 A records are appended with a TXT record. All other records are
 unhandled.
+
+.. _modes-of-operation-axfrend:
+
+Signaling end of an AXFR transfer for specific zone(s) using a script
+---------------------------------------------------------------------
+
+The PowerDNS Authoritative Server can invoke a Lua script on the of
+incoming AXFR zone transfer. The user-defined function ``axfr_end(zone)``
+within your script is invoked for each zone that has LUA-AXFR-END-SCRIPT
+defined in the ``domainmetadata``.
+
+What you can accomplish using a Lua script:
+
+- trigger a notification for a post processing event downstream
+- perform specific operation on the zone as a whole vs specific record/RRset
+
+To enable a Lua script for a particular slave zone, determine the
```

Comment:
Please don't use raw SQL. This can all be done via `pdnsutil`:

```
pdnsutil set-meta example.com LUA-AXFR-END-SCRIPT /lua/axfr-end.lua
```

Reply:
https://github.com/m0rcq/pdns/commit/159d92f1348e0c9b08ceb3d94fe79637ae48c81d addresses this.